helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/listener/HelpDeskRoomEventListener.java

```diff
@@ -166,7 +166,7 @@ public void onSymUserLeftRoom(SymUserLeftRoom symUserLeftRoom) {
         // Resend ticket message to the agent room
         SymMessage symMessage =
             SymMessageBuilder.message(String.format(MESSAGEML_TEMPLATE, question)).build();
-        symMessage.setFromUserId(localUser.getId());
+        symMessage.setFromUserId(ticket.getClient().getUserId());
```

Comment:
Why did you set the client id? Should it be the agent id?

Reply:
That's the bug. This information is used to fill the card information and the card was displaying the bot name as the customer name
```
public void sendTicketMessageToAgentStreamId(Ticket ticket, SymMessage message) {
...
SymUser symUser = symphonyClient.getUsersClient().getUserFromId(message.getFromUserId());
builder.username(symUser.getDisplayName());
builder.company(symUser.getCompany());
```